HHS/Head-Start-TTADP
# To Prod > 474 Move context text field above goals

## Description of change

- 474: Move context text field above goals.
- 485: (Backend only) add scope for reason to activity reports
- 490: (Backend only) add scope for target population to activity reports
- 494: Add grant and program specialist to grant list
- 489: Fix select box alignment on filters
- 501: Include goals with null statuses on goal selector
 
## How to test

- 474: Create a new AR the Context text field should now appear above the goals section.
- 485+490 - not in use yet
- 494: Confirm grant and program specialist are within grants list (recipient record)
- 489: Confirm all select box combinations are aligned with the filter panel
- 501: Create a goal in an AR. Create a second AR for the same Grantee, and confirm that this goal appears in the dropdown.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-474
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-485
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-490
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-489
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-494
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-501


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
